source/computer/editor.rst

```diff
@@ -0,0 +1,244 @@
+编辑器
+=======
+
+:本节贡献者: |姚家园|\（作者）、
+             |田冬冬|\（审稿）
+:最近更新日期: 2021-02-13
+:预计花费时间: 30 分钟
+
+----
+
+在终端中输入命令即可交给系统执行，但要编辑普通文件、脚本或者程序，则需使用一个叫做\ **编辑器**\ 的程序。
+编辑器是一个允许在屏幕上移动光标并编辑文字的程序，它是科研人员编写代码和和管理系统配置文件的重要工具。
+
+学习编辑器
+----------
+
+日常科研中，我们大部分时间都花在编写代码和普通文件上。因此，花点时间掌握某个适合自己的编辑器是非常必要的，
+也是十分值得的。学习和掌握一个新编辑器的步骤通常如下：
+
+- 阅读教程：例如，官方文档、本章教程等
+- 坚持使用新编辑器来完成所有的编辑工作：刚开始可能会让工作效率降低
+- 随时查阅：如果新编辑器的某个操作看起来像是有更方便的实现方法，一般情况下真的会有；如果觉得新编辑器
+  若具有一些功能可能会提高效率，一般情况下真的会有。
+
+按照上述步骤并坚持使用新的编辑器完成所有的编辑任务，则学习一个复杂的编辑器的过程通常是这样的：
+
+- 学习编辑器的基本操作（如前两个小时）：例如，打开和编辑文件、保存与退出等
+- 学习时间累计达到一定时间后（如 20 个小时），使用新编辑器的效率会和使用老编辑器一样快
+- 此后，新编辑器的益处开始显现。因为有了足够的知识和肌肉记忆后，使用新编辑器将大大节省时间
+
+现代编辑器都是一个复杂且强大的工具，永远有新东西可学：\ **学的越多，效率越高**\ 。
+
+选择编辑器
+----------
+
+当前存在许多优秀的编辑器，该使用哪个呢？这个问题并没有标准答案。选择其中一个一直用下去，遇到需要重复劳动或者不顺心的地方时，
+就去寻找解决办法，如更新配置和安装插件。有了足够多的知识和经验后，可以选择是否继续使用老编辑器，或尝试和更换新编辑器。
+
+按照操作方式，编辑器可分为两种类型：基于命令行的和图形化的。基于命令行的编辑器一般启动快，操作高效，学习成本较低；
+图形化编辑器启动慢一点，学习成本较高，但功能更加强大。
```

Comment:
> 基于命令行的编辑器一般启动快，操作高效，学习成本较低；图形化编辑器启动慢一点，学习成本较高，但功能更加强大。

I can't agree with you. Most CLI editors are difficult to learn.

Reply:
Done in eaaf45a.

> 按照操作方式，编辑器可分为两种类型：基于命令行的和图形化的。基于命令行的编辑器一般启动较快，操作高效；图形化编辑器启动慢一点，但功能更加强大。